### PR TITLE
Update app-management.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated WIP with purview related information

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/3102

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/apps/app-management.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Intune offers a range of capabilities to help you get the apps you need on the d
 | Apps from a   store | Yes | Yes | No | Yes |
 | Update apps | Yes | Yes | No | Yes |
 
-<sup>1</sup> Consider using [Windows Information Protection](../protect/windows-information-protection-configure.md) to protect apps on devices that run Windows 10/11.<br>
+<sup>1</sup> Consider using  [Microsoft Purview Information Protection](/microsoft-365/compliance/information-protection) and [Microsoft Purview Data Loss Prevention](/microsoft-365/compliance/dlp-learn-about-dlp). Purview simplifies the configuration set-up and provides an advanced set of capabilities. <br>
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-management.md:77]:
```suggestion
<sup>1</sup> Consider using  [Microsoft Purview Information Protection](/microsoft-365/compliance/information-protection) and [Microsoft Purview Data Loss Prevention](/microsoft-365/compliance/dlp-learn-about-dlp). Microsoft Purview simplifies the configuration set-up and provides an advanced set of capabilities. <br>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by msbemba** [comment]
@yogkumgit this has been approved and is ready to merge.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**